---
layout: default
---
			
<!-- Main -->
	<section class="wrapper style1">
		<div class="container">
			<div id="content">

				<!-- Content -->
			
					<article>
						
						<h3>SFyNCS</h3>
						<p>"SFyNCS" detects fusions of both protein-coding genes and non-coding sequences from RNAseq data. Click <a href="https://github.com/yanglab-computationalgenomics/SFyNCS">here</a> to download and <a href="https://biorxiv.org/cgi/content/short/2023.04.03.535462v1">here</a> to learn more.</p>
						
						<h3>HYENA</h3>
						<p>"HYENA" identifies candidate oncogenes activated by enhancer hijacking based on somatic SVs and gene expression data. Click <a href="https://github.com/yanglab-computationalgenomics/HYENA">here</a> to download and <a href="https://biorxiv.org/cgi/content/short/2023.01.09.523321v1">here</a> to learn more.</p>
						
						<h3>Starfish</h3>
						<p>"Starfish" infers complex genome rearrangement sigtures based on somatic SV and CNV patterns. It can assign individual complex rearrangement event to one of the six signatures identified from PCAWG. Click <a href="https://github.com/yanglab-computationalgenomics/Starfish">here</a> to download and <a href="https://www.nature.com/articles/s43018-022-00404-y">here</a> to learn more.</p>
						
						<h3>Beehive</h3>
						<p>"Beehive" is designed to reconstruct tumor evolutionary history based on somatic mutations. The ploidy for each chromosome and tumor purity have to be inferred prior to using this tool. Beehive can also help to validate the inferred purity and ploidy. Click <a href="https://zenodo.org/record/3358081">here</a> to download and <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1782-4">here</a> to learn more.</p>
						
						<h3>Meerkat</h3>
						<p>"Meerkat" is a computational tool to identify genomic rearrangements from high-throughput sequencing data. It uses multiple clusters of discordant reads to identify more complex rearrangements than available methods and uses split-reads to ensure high accuracy, followed by analysis of sequence homology at the breakpoints. The mechanisms forming these rearrangements can be inferred based on the features of breakpoints. Click <a href="https://github.com/guru-yang/Meerkat">here</a> or <a href="http://compbio.med.harvard.edu/Meerkat/">here</a> to download the software package. Click <a href="http://compbio.med.harvard.edu/Meerkat/Meerkat.example.v2.tar.gz">here</a> to download a demo dataset. Click <a href="http://www.cell.com/cell/fulltext/S0092-8674(13)00451-0">here</a> to learn more.</p>
					</article>
		
			</div>
		</div>
	</section>			
